internal/services/newrelic/new_relic_monitor_resource.go

```diff
@@ -520,3 +532,34 @@ func flattenUserInfoModel(input *monitors.UserInfo) []UserInfoModel {
 
 	return append(outputList, output)
 }
+
+// Currently the API only supports `SystemAssigned` type.
```

Comment:
If the Swagger spec specifies that System and User assigned should be supported but the API in reality only supported SystemAssigned then we should raise an issue on the Rest API specs repo.

Reply:
created a issue to track this problem: https://github.com/Azure/azure-rest-api-specs/issues/29256